docs/api/ReactWrapper/invoke.md

```diff
@@ -0,0 +1,41 @@
+# `.invoke(propName)(...args]) => Any`
```

Comment:
```suggestion
# `.invoke(propName) => (...args) => Any`
```

Reply:
@ljharb `invoke` is the same interface with `renderProp`, which is `.renderProp(key)() => ShallowWrapper`. Why is this change needed?
(`]` is a typo so I'll remove this though)

https://airbnb.io/enzyme/docs/api/shallow.html#renderpropkey--shallowwrapper

Reply:
true, fair point. match renderProp for now; if i want to change them i'll change both at once later.